### ISSUE TITLE
Battery Level

### ISSUE THREAD

**Issue by flymac** [Airthings/wave-reader#11]
Hi,
Would it be possible to also display the battery level of the device? Since the App displays it, I guess it should also be one of the returned values ?
Thanks!

**Comment by AndySchroder** [comment]
yes, would like this, or a model that allows power to be hard wired.

**Comment by baudfather** [comment]
I'd also like to know if this is possible, very valuable for use in integrating.
